go/vt/vttablet/tabletmanager/action_agent.go

```diff
@@ -238,6 +244,7 @@ func NewActionAgent(
 	// Create the TabletType stats
 	agent.exportStats = true
 	agent.statsTabletType = stats.NewString("TabletType")
+	agent.statsTabletTypeCount = stats.NewCountersWithSingleLabel("TabletTypeCount", "Number of state change to the labeled topodata.TabletType", "type")
```

Comment:
nit "Number of times the tablet changed to the given type"

Reply:
Changed it to: "Number of times the tablet changed to the labeled type"